pallets/xor-fee/src/lib.rs

```diff
@@ -1264,10 +1268,18 @@ pub mod pallet {
     #[pallet::event]
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
     pub enum Event<T: Config> {
+        #[cfg(feature = "wip")] // Xorless fee
         /// Fee has been withdrawn from user. [Account Id to withdraw from, Asset Id to withdraw, Fee Amount]
         FeeWithdrawn(AccountIdOf<T>, AssetIdOf<T>, Balance),
+        #[cfg(feature = "wip")] // Xorless fee
         /// The portion of fee is sent to the referrer. [Referral, Referrer, AssetId, Amount]
         ReferrerRewarded(AccountIdOf<T>, AccountIdOf<T>, AssetIdOf<T>, Balance),
+        #[cfg(not(feature = "wip"))] // Xorless fee
+        /// Fee has been withdrawn from user. [Account Id to withdraw from, Fee Amount]
+        FeeWithdrawn(AccountIdOf<T>, Balance),
+        #[cfg(not(feature = "wip"))] // Xorless fee
+        /// The portion of fee is sent to the referrer. [Referral, Referrer, Amount]
+        ReferrerRewarded(AccountIdOf<T>, AccountIdOf<T>, Balance),
```

Comment:
If we change the structure of event - it could broke indexers.
Probably it is better to add new events with new structure and just keep deprecated events for backward compatibility

Reply:
Yeah, that's why cover with wip.
Going update indexer. We don't want add separate events.
And asset id even now used in indexer, but only for xor.